src/widgets/switch/lv_switch.h

```diff
@@ -39,6 +49,28 @@ LV_ATTRIBUTE_EXTERN_DATA extern const lv_obj_class_t lv_switch_class;
  */
 lv_obj_t * lv_switch_create(lv_obj_t * parent);
 
+/*=====================
+ * Setter functions
+ *====================*/
+
+/**
+ * Set the orientation of switch.
+ * @param obj           pointer to switch object
+ * @param orientation   switch orientation from `lv_switch_orientation_t`
+ */
+void lv_switch_set_orientation(lv_obj_t * obj, lv_switch_orientation_t orientation);
+
+/*=====================
+ * Getter functions
+ *====================*/
```

Comment:
remove

Reply:
Hi, I don't quite understand what you mean. I don't think these should be removed, refer to: https://github.com/lvgl/lvgl/blob/master/src/widgets/bar/lv_bar.h#L37-L41

Reply:
> Hi, I don't quite understand what you mean. I don't think these should be removed, refer to: https://github.com/lvgl/lvgl/blob/master/src/widgets/bar/lv_bar.h#L37-L41

OK